plugins/optionrom/fu-optionrom-plugin.c

```diff
@@ -43,10 +42,27 @@ fu_optionrom_plugin_startup(FuPlugin *plugin, FuProgress *progress, GError **err
 	return TRUE;
 }
 
+static gboolean
+fu_optionrom_plugin_device_created(FuPlugin *self, FuDevice *device, GError **error)
+{
+	if (!fu_device_probe(device, error))
+		return FALSE;
+	if (!fu_device_has_flag(device, FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE)) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_NOT_SUPPORTED,
+				    "unable to read firmware from device, 'rom' does not exist");
+		return FALSE;
+	}
```

Comment:
this should be unnecessary and dead code - the same thing will happen when running `fu_optionrom_device_probe()` afaict.